### DISCUSSION TITLE
Is annotations supported in scripted dashboard?

### DISCUSSION THREAD

**Discussion by N3v3R3nD** [grafana/grafana#39589] [Dashboards, Panels & Visualizations]
Is it possible to show the annotations on a scripted dashboard?
I can not get it to show any when I've tried, the option for the annotations does not show the once in the script.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
